usr/share/rear/prep/GNU/Linux/240_include_multipath_tools.sh

```diff
@@ -1,7 +1,9 @@
 # 240_include_multipath_tools.sh
 # Boot Over SAN executables and other goodies
 
-[[ $BOOT_OVER_SAN != ^[yY1] ]] && return
+if ! is_true "$BOOT_OVER_SAN" ; then
```

Comment:
I just wonder what happen if `BOOT_OVER_SAN` was not set ... will it be treated as False ?